create_tag.py

```diff
@@ -174,8 +174,8 @@ def get_pullrequest_infos(args, repo, hashes):
                 msg = (f"- {pr_title_line}\n"
                        f"  - {author_reviewers_line}")
             else:
```

Comment:
not sure why there is this if/else - but this looks pretty obsolete now

Reply:
I am not familiar with the backstory on why the indent level is different between `cockpit-composer` and all other projects; I have a followup PR that rewrites the if/else to use `textwrap.indent` instead.

Reply:
I've added a commit to do this.